### PR TITLE
S3 Orchessstration 🐍

### PR DESCRIPTION
**Pull request by mikegrima**
- See #5 for details.

### CODE REVIEW

File: cloudaux/aws/s3.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def get_bucket_region(**kwargs):
     try:
         result = get_bucket_location(**kwargs)
         location = result['LocationConstraint']
-    except ClientError, e:
```
Comment (mikegrima) [inline cloudaux/aws/s3.py]: PyCharm complained about this not working for Python 3.5.

Comment (monkeysecurity) [inline cloudaux/aws/s3.py]: does python 3 require it to say `except ClientError as e:` ?

Comment (monkeysecurity) [inline cloudaux/aws/s3.py]: oh, I see your update now. :) cool

**Comment by monkeysecurity** [comment]
We'll want to update the `README.md` section on s3 too.

**Pull request merged**